modules/Microsoft.CognitiveServices/accounts/.test/common/deploy.test.bicep

```diff
@@ -63,6 +63,7 @@ module testDeployment '../../deploy.bicep' = {
     kind: 'Face'
     customSubDomainName: '<<namePrefix>>xdomain'
     diagnosticLogsRetentionInDays: 7
+    enableEncryption: false
```

Comment:
So do I get this right that the implementation is now aligned but we still miss prerequisites to test it in our own environment? Would that be the case also for the MS managed keys encryption?

Reply:
No-ish.
When I enable encryption without a CMK I actualy do get an error along the lines of 'this not not yet available' or so. However, when I check the [docs](https://learn.microsoft.com/en-us/azure/cognitive-services/computer-vision/identity-encrypt-data-at-rest#about-cognitive-services-encryption) it reads like Service-Managed encryption is anyways enabled.

So what I did now was to align it even closer to the other implementation in that you only have the choice of CMK - or not. Without any additional implementation for Service-Managed keys.

While cleaning up the boolean I noticed also that some of our implementation use 'allowedValues' for a bool property and removed those too. It's just very few files fortunately.